src/lib/util/value.c

```diff
@@ -1527,65 +1541,101 @@ ssize_t fr_value_box_from_network(TALLOC_CTX *ctx,
 			.af = AF_INET,
 			.prefix = 32,
 		};
-		memcpy(&dst->vb_ip.addr.v4, p, (end - p));
+		fr_dbuff_out_memcpy((uint8_t *)&dst->vb_ip.addr.v4, &work_dbuff, len);
 		break;
 
 	case FR_TYPE_IPV4_PREFIX:
 		dst->vb_ip = (fr_ipaddr_t){
 			.af = AF_INET,
-			.prefix = *p++,
 		};
-		memcpy(&dst->vb_ip.addr.v4, p, (end - p));
+		fr_dbuff_out(&dst->vb_ip.prefix, &work_dbuff);
+		fr_dbuff_out_memcpy((uint8_t *)&dst->vb_ip.addr.v4, &work_dbuff, len - 1);
```

Comment:
All these should use the RETURN variant

Reply:
Agreed.

Reply:
Corrected and pushed.

Reply:
Remaining fr_dbuff_out* converted to the _RETURN versions and pushed.